app/bootstrap.php

```diff
@@ -27,5 +27,17 @@
     ->useAutowiring(false)
     ->build();
 
+if (!file_exists('/usr/local/bin')) {
```

Comment:
good call... I'll check to make sure the results are right in Cygwin, if they are I'll go with that approach.